qt-models/diveplannermodel.cpp

```diff
@@ -894,8 +894,8 @@ void DivePlannerPointsModel::createPlan(bool replanCopy)
 				add_dive_to_trip(copy, current_dive->divetrip);
 			record_dive(copy);
 			QString oldnotes(current_dive->notes);
```

Comment:
I'm meanwhile not sure any more if I understand what lines 896 to 900 are supposed to do. And I'm also not sure if it works...